package.json

```diff
@@ -882,6 +882,7 @@
     "email-addresses": "^5.0.0",
     "execa": "^4.0.2",
     "expiry-js": "0.1.7",
+    "exponential-backoff": "^3.1.1",
```

Comment:
Kibana already has this dependency as a transitive dep, thought it made sense to just use it again here.